endpoint/labels.go

```diff
@@ -104,3 +118,21 @@ func (l Labels) Serialize(withQuotes bool) string {
 	}
 	return strings.Join(tokens, ",")
 }
+
+func (l Labels) Serialize(withQuotes bool, txtEncryptEnabled bool, aesKey []byte) string {
+	if !txtEncryptEnabled {
+		return l.SerializePlain(withQuotes)
+	}
+	text := l.SerializePlain(false)
+	log.Debugf("Encrypt the serialized text %#v before returning it.", text)
+	var err error
+	text, err = EncryptText(text, aesKey)
+	if err != nil {
+		log.Debugf("Failed to encrypt the text %#v using the encryption key %#v. Got error %#v.", text, aesKey, err)
+	}
```

Comment:
This error is masked. This function should probably return an error.

Reply:
@Raffo there is a little to do with this mr
@vsychov do you have working fork with txt encryption feature?

Reply:
@shcherbak, yes, I'm using this feature in a production with the Cloudflare provider since the moment of the PR creation. If you want I can publish a working Docker image for your convenience.

Reply:
@vsychov this is amazing, thank you
but i think will wait for official chart and image
nicely done, respect